java/packagescanner.go

```diff
@@ -72,7 +72,7 @@ type Scanner struct {
 func (*Scanner) Name() string { return "java" }
 
 // Version implements scanner.VersionedScanner.
-func (*Scanner) Version() string { return "7" }
+func (*Scanner) Version() string { return "8" }
```

Comment:
I just updated this from 6 to 7 in https://github.com/quay/claircore/pull/1467. Depending on when this is merged, I can revert this